Personas.md

```diff
@@ -14,8 +14,6 @@ benefited from the project.*
 
 ## Persona - The PhD student
 
-![right-aligned-image](/assets/phd.jpg)
```

Comment:
Not sure why these image where removed, but as https://www.ewatercycle.org/Personas.html can only be found directly (its disabled in menu) it is probably OK.

Reply:
These are copyrighted images, and the page is not shown, so we ended up removing them.